packaging/greenboot/microshift-running-check.sh

```diff
@@ -25,6 +25,33 @@ function forced_termination() {
     exit 1
 }
 
+# Check preconditions for existence of topolvm deployment.
+# Adapted from MicroShift code.
+#
+# args: None
+# return: 0 if topolvm readiness should be checked, 1 otherwise
+function topolvmShouldBeDeployed() {
+    if ! hash vgs 2>/dev/null; then
+        return 1
+    fi
+
+    if [ -f /etc/microshift/lvmd.yaml ]; then
+        return 0
+    fi
```

Comment:
@ggiguash 
This has one caveat: in the code, after loading the file, we also check if `len(DeviceClass) > 0`. Should we skip this level of detail? Or do you have idea how to easily get this info with using standard linux tools without `yq`

Reply:
I think we can skip this detail. It's too much for this script.